Pod/Classes/Cards/Card.swift

```diff
@@ -12,6 +12,9 @@ import Foundation
  A card represents a physical bank card with all its associated attributes.
 */
 public struct Card {
+    
+    // The card franchise name.
+    public let franchise: Franchise
```

Comment:
Is there a reason not to use the existing `CardType`? An enum might not work when using custom card types that are not in this enum.